components/brave_extension/extension/brave_extension/background/today/feed.ts

```diff
@@ -13,6 +13,7 @@ type RemoteData = BraveToday.FeedItem[]
 
 type FeedInStorage = {
   storageSchemaVersion: number,
+  sourceUrl: string,
```

Comment:
I see both `Url` and `URL`. Is there a preference either way?

Reply:
I don't have a preference, I don't think we have a standard in our JS. On the chromium side it looks like it'd be `sourceUrl` and `getRegionUrlPart`, so I'll change it to that.

Reply:
dang - Chromium is super inconsistent with this. Looks like they use lowercase for namespaces + file names with URL, but functions, variable, and class names are a mix between `Url` and `URL`. Whatever we pick, I'd just say let's be consistent 😄

Reply:
Their naming standard for both JavaScript and c++ states `Url` so I'll use that.